custom-namespaces.yaml

```diff
@@ -423,26 +390,11 @@ firefox_accounts:
         - channel: release
           table: mozdata.firefox_accounts.fxa_log_device_command_events
   explores:
-    all_event_counts:
-      type: events_explore
-      views:
-        base_view: all_events
-        extended_view: all_events_table
     event_counts:
       type: events_explore
       views:
         base_view: events
-        extended_view: fxa_events_table
-    auth_event_counts:
-      type: events_explore
-      views:
-        base_view: auth_events
-        extended_view: fxa_auth_events_table
-    content_auth_stdout_event_counts:
-      type: events_explore
-      views:
-        base_view: content_auth_stdout_events
-        extended_view: fxa_content_auth_stdout_events_table
```

Comment:
According to the data catalog there are [134 Looker charts using the `firefox_accounts.content_auth_stdout_event_counts` explore](https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:looker,firefox_accounts.explore.content_auth_stdout_event_counts,PROD)/Lineage), which will break if the explore is removed.